### ISSUE TITLE
Accessibility for combobox

### ISSUE THREAD

**Issue by rsukale** [salesforce/design-system-react#1626]
At trailhead, we use the combobox as in our search bar to show search suggestions.
We had a VPAT audit a while ago and were given the following feedback

"Once the focus is set on the search text box, screen reader does not announce any instructions for assistive technology users as to how they can perform search with the use of “Up and Down” arrow keystrokes, to navigate within the results. "

I think it would be useful if this announcement by DSR itself, but I am not sure how to go about it.

**Comment by interactivellama** [comment]
@rsukale Do you all have an a11y specialist assigned to your product? It's got a menu list box attached to up and down arrows would be implied due to `aria-haspopup="listbox"` on the input.

I would suggest asking them sending a link to what's missing from this page. https://www.lightningdesignsystem.com/components/combobox/#Entity-Autocomplete-Combobox-(Lookup)

**Comment by jiahanericsson-salesforce** [comment]
@interactivellama hi Stephen, what are your thoughts on the feedback from Accessibility? Do you think  the total results count could be something to incorporate? Thanks for your time!

**Comment by interactivellama** [comment]
I'm hesitant to promote implementing a non-SLDS pattern, I've reached out to the SLDS and Accessibility teams internally for comment on GUS.

**Comment by interactivellama** [comment]
Based on internal conversation, this is not part of the Combobox ARIA spec http://w3c.github.io/aria-practices/#combobox nor the SLDS implementation https://www.lightningdesignsystem.com/components/combobox/#Entity-Autocomplete-Combobox-(Lookup)

Please re-open this issue if SLDS Combobox changes.
